pkg/controller/secrets/secret_test.go

```diff
@@ -108,8 +110,8 @@ func TestBasic_Gen(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	assert.Len(t, resp.Client.Created, 2)
-	assert.Len(t, resp.Collected, 2)
+	require.Len(t, resp.Client.Created, 2)
```

Comment:
Asking because I don't know: why is this better?

Reply:
This fails the testcase before it gets into an index out-of-bounds issue in a check below.